### PR TITLE
Updated protogeometry icons that were missing from library

### PR DESCRIPTION
**Pull request by Racel**
Fixing https://jira.autodesk.com/browse/DYN-857
Some icons did not exist and some were misspelled. 

Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

Update icons that were messed up in new library. These were mainly protogeometry nodes that did not have icons or there was a misspelling in the name.

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [x] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@alfarok

### CODE REVIEW

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs [hunk 1]
```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("2.0.0.2821")]
+[assembly: AssemblyVersion("2.0.0.3416")]
```
Comment (mjkkirschner) [inline src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs]: please revert this file.

File: src/Libraries/GeometryUIWpf/GeometryUIWpf.csproj [hunk 2]
```diff
@@ -127,6 +127,27 @@
     </EmbeddedResource>
     <EmbeddedResource Include="ProtoGeometryImages.resx" />
   </ItemGroup>
+  <ItemGroup>
```
Comment (mjkkirschner) [inline src/Libraries/GeometryUIWpf/GeometryUIWpf.csproj]: seems incorrect that these files are embedded directly in the proj file - aren't the rest of them just embedded into the resx? could be wrong here.

Comment (alfarok) [reply src/Libraries/GeometryUIWpf/GeometryUIWpf.csproj]: @Racel Did you embed the resources you added as well?  You should only have to embed the icon after it is added to the resx not the original resources.

Comment (alfarok) [reply src/Libraries/GeometryUIWpf/GeometryUIWpf.csproj]: To clarify:
- add the resources
- drag them into the appropriate .resx
- select the icon you just drop in the .resx (not the original resource)
- embed in the properties window

**Comment by Racel** [comment]
Can I just close the PR and start over? Looks like I didn't update the right resx file, etc....

File: src/Libraries/GeometryUIWpf/GeometryUIWpf.csproj (hunk 2 above)
Comment (Racel) [reply src/Libraries/GeometryUIWpf/GeometryUIWpf.csproj]: @alfarok - i guess i was embedding the resource, but not sure how to select just the icon...

**Pull request closed without merging**